test/WebJobs.Script.Tests/Binding/HttpBindingTests.cs

```diff
@@ -109,147 +73,36 @@ public void ParseResponseObject_StatusWithNullBody_ReturnsExpectedResult()
 
             object content = null;
             IDictionary<string, object> headers = null;
-            HttpStatusCode statusCode = HttpStatusCode.OK;
+            int statusCode = StatusCodes.Status200OK;
             bool isRawResponse = false;
             HttpBinding.ParseResponseObject(responseObject, ref content, out headers, out statusCode, out isRawResponse);
 
             Assert.Equal(null, content);
-            Assert.Equal(HttpStatusCode.Accepted, statusCode);
-        }
-
-        [Fact]
-        public async Task CreateResultContent_ExpandoObject_ReturnsJsonStringContent()
```

Comment:
No longer applicable. It would be good to add a test to validate the method to get result, but I'm trying to keep the scope limited to re-enabling the existing tests we were missing.